pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1004,6 +1004,8 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			addVirtualMachine(vmi)
 
 			controller.Execute()
+			Expect(virtClientset.Actions()).To(HaveLen(1)) //1 is the vmi creation
```

Comment:
We should be able to do even better and be able to check what action. Meaning that we can code in that only the vmi creation happened.